pouta_blueprints/static/js/controllers/StatsController.js

```diff
@@ -0,0 +1,10 @@
+app.controller('StatsController', ['$q', '$scope', '$http', '$interval', '$uibModal', 'AuthService', 'Restangular',
+                              function ($q,   $scope,   $http,   $interval,   $uibModal,   AuthService,   Restangular) {
+
+        Restangular.setDefaultHeaders({token: AuthService.getToken()});
+        var stats = Restangular.all('stats');
+
+        stats.getList().then(function (response) {
+            $scope.stats = response[0];
```

Comment:
Marshalling should return response that can directly be inserted into $scope.stats
$scope.stats = response[0]; -> $scope.stats = response;